.github/workflows/build.yml

```diff
@@ -21,6 +24,27 @@ jobs:
         run: yarn install --frozen-lockfile
       - name: Build
         run: yarn build
+      - name: Setup Apple Code Signing
```

Comment:
I see this adds quite a bit of runtime to the build step. Can we split this off into its own job so that other jobs that don't depend on it can already start? 

The nice thing about the wizard is that CI is relatively fast and I'd be happy if it stays fast :D

Reply:
This should not block unit tests but it _rightfully_ should block e2e tests as the tests are run on these binaries. I'll see if there's improvements I can make.